File: Class-2.4/src/org/launchcode/main/Message.java
package org.launchcode.main;

public class Message {
	
	public static String getMessage(String lang) {
		if (lang.equals("sp")) {
			return "¡Hola, Mundo!";
		} else if (lang.equals("fr")) {
			return "Bonjour, le monde!";
		} else {
			return "Hello, World!";
		}
	}
}

File: CSharp/LC101-Unit2/Class-2.1/HelloMethods/Message.cs
namespace HelloMethods
{
    public class Message
    {
        public static string GetMessage(string lang)
        {
            if (lang.Equals("sp"))
            {
                return "Hola Mundo";
            }
            else if (lang.Equals("fr"))
            {
                return "Bonjour le monde";
            }
            else
            {
                return "Hello World";
            }
        }


        private static string GetPrivateMessage(string hi)
        {
            return "Secret message: " + hi;
        }
    }
}
